draft-gruessing-moq-requirements.md

```diff
@@ -126,22 +126,29 @@ This may help to explain why interactive use cases have typically relied on prot
 |**Senders/Receivers**|  One to One
 |**Bi-directional**| Yes
 
-Where media is received, and user inputs are sent by the client. This may also
-include the client receiving other types of signaling, such as triggers for
-haptic feedback. This may also carry media from the client such as microphone
-audio for in-game chat with other players.
+In this use case the computation for running a video game (single or
+multiplayer) is performed externally on a hosted service, with user inputs from
+input devices sent to the server, and video and audio of gameplay returned. This
+may also include the client receiving other types of signaling, such as triggers
+for haptic feedback, as well as media from the client such as microphone audio
+for in-game chat with other players. Latency may be considerably important in
+this use case as updates to video occur in response user input, with certain
+genres of games having high requirements in responsiveness and/or a high
+frequency of user input.
 
 ### Remote Desktop {#remdesk}
 
 |Attribute | Value |
 | -----+----------------
-|**Senders/Receivers**|  One to One
+|**Senders/Receivers**|  One to Many
 |**Bi-directional**| Yes
 
-Where media is received, and user inputs are sent by the client. Latency
-requirements with this use case are marginally different than the gaming use
-case. This may also include signalling and/or transmitting of files or devices
-connected to the user's computer.
+Similar to the gaming use case, but where a user wishes to observe or control
```

Comment:
Perhaps "Similar to the gaming use case in many requirements", just to get us out of trading picked nits with other WG members.